plugin.video.devour.com/addon.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.devour.com" version="1.0.0" name="Devour" provider-name="lambda">
+<addon id="plugin.video.devour.com" version="1.1.0" name="Devour" provider-name="lambda">
```

Comment:
I thought about it too, but I was undecided about it.
Uncrate and Devour were too different sites owned by UNCRATE, LLC
Uncrate is a magazine for men http://uncrate.com/magazine/
I am undecided because if it is renamed to Uncrate, users that know about the magazine will think that this is an add-on with videos for men.
They keep name this part as Uncrate Devour Videos, probably because it doesn't have content only for men. 
http://uncrate.com/about/